app/lib/resources.server.ts

```diff
@@ -80,7 +110,11 @@ export async function getResource(
     throw new Error(`Could not find GitHub data for ${resource.repoUrl}`);
   }
 
-  return { ...resource, ...gitHubData, readmeHtml };
+  return {
+    ...resource,
+    ...gitHubData,
+    readmeHtml: replaceRelativeLinks(readmeHtml, resource.repoUrl),
```

Comment:
Let's move the `replaceRelativeLinks` inside `fetchReadme`, since the output of `fetchReadme` is server-side cached in the LRU cache (`resourceReadmeCache`). Otherwise we're going to do this replacement every time the resources is requested, instead of just the first time (every 5 minutes).

This should do it

```
let { html } = await processMarkdown(md);
return replaceRelativeLinks(html, `https://github.com/${key}`);
```

Feel free to pull `https://github.com/` out as a constant and reference that, since we use it like 3 places. No big deal if you don't want to do that step

Reply:
good point here!